scoring.js

```diff
@@ -0,0 +1,81 @@
+const calculateScore = (player) => {
+    switch (player.position) {
+      case 'QB':
+        return calculateQBScore(player)
+      case 'RB':
+        return calculateRBScore(player)
+      case 'WR':
+        return calculateWRScore(player)
+      case 'TE':
+        return calculateTEScore(player)
+      default:
+        return 0
+    }
+  }
+  const calculateQBScore = (player) => {
+    let score = 0
+    const passing = player.stats.passing
+    const rushing = player.stats.rushing
+  
+    score += passing.yards / 25
+    score += passing.touchdowns * 6
+    score -= passing.interceptions * 3
+    score += rushing.yards / 10
+    score += rushing.touchdowns * 6
+    score -= rushing.fumbles * 3
+  
+    return score
+  }
+  const calculateRBScore = (player) => {
+    let score = 0
+    const rushing = player.stats.rushing
+    const receiving = player.stats.receiving
+  
+    score += rushing.yards / 10
+    score += rushing.touchdowns * 6
+    score -= rushing.fumbles * 3
+    score += receiving.receptions * 1
+    score += receiving.yards / 10
+    score += receiving.touchdowns * 6
+    score -= receiving.fumbles * 3
+  
+    return score
+  }
```

Comment:
Try and simplify code, for example calculateRBScore could also look like this

const calculateRBScore = (player) => {
  const rushing = player.stats.rushing
  const receiving = player.stats.receiving

  const score = rushing.yards / 10 +
   rushing.touchdowns * 6 -
   rushing.fumbles * 3 +
   receiving.receptions * 1 +
   receiving.yards / 10 +
   receiving.touchdowns * 6 -
   receiving.fumbles * 3

  return score
}